samples/nativeSample/src/main/java/com/inmobi/nativead/sample/listview/ListViewFeedFragment.java

```diff
@@ -86,10 +87,14 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
     private void createStrands() {
         for (int position : mAdPositions) {
-            final InMobiNative nativeStrand = new InMobiNative(getActivity(),
-                    PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener(position));
+            try {
+                final InMobiNative nativeStrand = new InMobiNative(getActivity(),
+                        PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener(position));
 
-            mStrands.add(nativeStrand);
+                mStrands.add(nativeStrand);
+            } catch (SdkNotInitializedException e) {
```

Comment:
`try/catch` block is not necessary for `SdkNotInitializedException` as the intention is to throw exception only for wrong integration. Publishers following the guidelines need not even handle this exception as it will never occur.